modules/graceful-restart.adoc

```diff
@@ -64,7 +64,31 @@ $ oc describe csr <csr_name> <1>
 $ oc adm certificate approve <csr_name>
 ----
 
-. After the control plane nodes are ready, verify that all worker nodes are ready.
+. After the control plane nodes are ready, mark all the nodes in the cluster as schedulable. You can do this from your cloud provider's web console, or by running the following loop:
```

Comment:
2 comments here:

- IMHO we should uncordon after checking that both control plane nodes and worker nodes are ready, not checking the control planes, then uncordon, then check worker nodes.
- Nodes are not uncordoned "from your cloud provider's web console" but from the OpenShift console. The cloud provider's console (e.g. AWS web console, Azure Portal, vSphere web client...) are intended to manage the cloud provider resources (VMs, virtual disks...) and know nothing about OpenShift.